.rubocop.yml

```diff
@@ -13,6 +14,7 @@ AllCops:
   Exclude:
     - 'bin/*'
     - 'gemfiles/*'
+    - 'vendor/bundle/**/*'
```

Comment:
I'd broaden this to `'vendor/**/*'`. We should never be linting any vendored thing we ever put in there, not just the bundle cache